README.md

```diff
@@ -24,6 +24,7 @@ This repo contains subgraphs defined using a set of standardized schemas. These
 - Implement the mappings, deploy and test your data using either Hosted Service or The Graph Studio.
 - For metrics calculation (e.g. revenue, fees, TVL), please refer to the `README.md` in the protocol's subgraph folder for methodology. There is also a broader explanation of how different fields are defined in the schema in `docs/Schema.md`. Feel free to reach out to me if anything isn't clear.
 - We've built a handy debugging/validation dashboard for you to quickly visualize the data in your subgraph. It's deployed to [subgraphs.xyz](https://subgraphs.xyz/) and the source code is under `dashboard` if you want to spin it up locally.
+- There is also a cli-tool called `messari-cli`. It utilized The Graph CLI and exploits the standard directory structure to template and prepare subgraph deployments for you with simple commands. You can install the CLI by moving into the `messari-cli` folder and running `npm install -g ./` to install the package then `npm install` to make sure all node modules used by the package are installed properly. This installs locally by creating a reference to this package on your machine. Therefore, if the location of your local copy of the repo changes, you will want to unintall this version and reinstall in the new location. Try running `messari b/build` to be prompted for building a subgraph. Try `messari b/build -d` to be prompted to build and then deploy subgraphs. Run `messari b/build -h` to be shown a list of optional paramters you can use to bypass prompts or add extra detail about the build/deployment you would like to execute. You can find more information about the CLI in the `messari-cli/docs/` folder.
```

Comment:
imo you should move the explanation of how to install and use into messari-cli/readme.md

Reply:
Agree 👍 

A short comment here and a link to a more detailed README inside messari-cli might be best

Reply:
Alright. Updated - Let me know how it is!